packages/fxa-settings/src/components/App/RememberPassword/en.ftl

```diff
@@ -0,0 +1,5 @@
+## RememberPassword component
+
+# Link that users can follow to sign in to their account
+# This link exits the Reset Password flow
+pw-remember-pw = Remember password? Sign in
```

Comment:
Doesn't this sound a bit strange (mechanical) without "your"?

Reply:
It does. I'm not sure if there was a reason why it's written this way... @LZoog, can I go ahead and change this to "Remember your password? Sign in", or should I check in with someone (content?)?

Reply:
@vpomerleau I'd double check with Vesta, she can point you to the right person if it's not up to her.